test/helpers/SubscriptionJsonFixtures.scala

```diff
@@ -0,0 +1,447 @@
+          )
+          )),
+        "requestDetail" -> Json.obj(
+          "idType" -> "idType",
+          "idNumber" -> "idNumber",
+          "isGBUser" -> true,
+          "primaryContact" -> Json.obj(
+            "individual" -> Json.obj(
+              "firstName" -> "Fairy",
+              "lastName" -> "Liquid"
+            ),
+            "email" -> "email2@email.com",
+            "phone" -> "01910002222",
+            "mobile" -> "07500000000"
+          )
+        )
+      )
+    )
+
+  //Organisation Fixtures
+  val jsonPayloadForOrganisation: String =
+    """
+      |{
+      |  "createSubscriptionForDACRequest": {
+      |    "requestCommon": {
+      |      "regime": "DAC",
+      |      "receiptDate": "2020-09-23T16:12:11Z",
+      |      "acknowledgementReference": "AB123c",
+      |      "originatingSystem": "MDTP",
+      |      "requestParameters": [{
+      |        "paramName":"Name",
+      |        "paramValue":"Value"
+      |      }]
+      |    },
+      |    "requestDetail": {
+      |      "idType": "idType",
+      |      "idNumber": "idNumber",
+      |      "isGBUser": true,
+      |      "primaryContact": {
+      |        "organisation": {
+      |          "organisationName": "Pizza for you"
+      |        },
+      |        "email": "email@email.com",
+      |        "phone": "0191 111 2222",
+      |        "mobile": "07111111111"
+      |      }
+      |    }
+      |  }
+      |}
+      |""".stripMargin
+
+  val organisationSubscription: CreateSubscriptionForDACRequest =
+    CreateSubscriptionForDACRequest(
+      SubscriptionForDACRequest(
+        RequestCommonForSubscription(
+          "DAC",
+          "2020-09-23T16:12:11Z",
+          "AB123c",
+          "MDTP",
+          Some(Seq(RequestParameters(
+            "Name",
+            "Value"
+          )))
+        ),
+        RequestDetail(
+          "idType",
+          "idNumber",
+          None,
+          true,
+          PrimaryContact(
+            ContactInformationForOrganisation(
+              OrganisationDetails("Pizza for you"),
+              "email@email.com",
+              Some("0191 111 2222"),
+              Some("07111111111"))
+          ),
+          None
+        )
+      )
+    )
+
+  val organisationSubscriptionJson =
+    Json.obj(
+      "createSubscriptionForDACRequest" -> Json.obj(
+        "requestCommon" -> Json.obj(
+          "regime" -> "DAC",
+          "receiptDate" -> "2020-09-23T16:12:11Z",
+          "acknowledgementReference" -> "AB123c",
+          "originatingSystem" -> "MDTP",
+          "requestParameters" -> Seq(Json.obj(
+            "paramName" -> "Name",
+            "paramValue" -> "Value"
+          ))
+        ),
+        "requestDetail" -> Json.obj(
+          "idType" -> "idType",
+          "idNumber" -> "idNumber",
+          "isGBUser" -> true,
+          "primaryContact" -> Json.obj(
+            "organisation" -> Json.obj(
+              "organisationName" -> "Pizza for you"
```

Comment:
can you shove a generator in for organisation names instead?
similarly for phone and mobile numbers

Reply:
it means these will have to be functions and the model fixtures too